doc/release/README.multilocale.rst

```diff
@@ -0,0 +1,233 @@
+============================
+Multilocale Chapel Execution
+============================
+
+To run Chapel on multiple locales, the following steps are required:
+
+
+0. The following information describes a fairly generic use of
```

Comment:
Step "0" is more of a pre-step, so I opted to keep it as 0.  The lists in this file do use auto-numbering, but to start with something other than the default of "1." you need to specify the first number/letter/roman numeral.